src/hooks/resetScroll/resetScroll.hook.jsx

```diff
@@ -0,0 +1,13 @@
+import { useRef, useEffect } from 'react';
+
+const useResetScroll = () => {
+  const resetScrollRef = useRef(null);
+  useEffect(() => {
+    if (resetScrollRef.current) {
+      resetScrollRef.current.scrollLeft = 0;
+      resetScrollRef.current.scrollTop = 0;
+    }
+  });
```

Comment:
Nice hook, but if you add some interactions that trigger the rerender the hook will be called again and it will scroll to the top. I can see on the working app why did you scroll to top-left, but I recommend using the depency with `[]` so it should be triggered only the firs time